### DISCUSSION TITLE
Does the tool plan to support the new Dual-Write story?

### DISCUSSION THREAD

**Discussion by princzkeljanos** [microsoft/Dual-write-automations#38] [Q&A]
Could you let me know if you plan to adjust this tool to the new Dual Write experience?
https://www.yammer.com/dynamicsaxfeedbackprograms/#/files/1603169050624 - Go to 12 min; the demo starts there.

Or will it be retired? I just want to know if we build any automation on this, can we use it forward until it is not upgraded to solution methodology, or can we throw it away after the new Dual Write Experience becomes GA?

Thanks.

**Comment by Hax1337** [comment]
I do not know what the extend will be but based on the presentation this tool won't be needed anymore once the solution aware solution is in place. 

However, there will be still a longer time until all customers have upgraded to this approach if there is a hybrid period. If thats the case customers using the "old" solution are still benefiting from this tool.
